definitions/checks/foreman/check_https_proxies.rb

```diff
@@ -0,0 +1,35 @@
+module Checks
+  class CheckHttpsProxies < ForemanMaintain::Check
+    metadata do
+      label :https_proxies
+      for_feature :foreman_database
+      description 'Check for HTTPS proxies from DB'
+      manual_detection
+    end
+
+    def run
+      https_proxies = find_https_proxies
+      unless https_proxies.empty?
+        https_proxy_names = https_proxies.map { |proxy| proxy['name'] }
+        question = "Syncing repositories through an \'HTTP Proxy\' that uses the https:// "\
+                    "protocol is not supported directly with Satellite 6.10.\n"\
+                    "The following proxies use HTTPS: #{https_proxy_names.join(',')}.\n"\
+                    "For a workaround see https://access.redhat.com/solutions/6414991\n"\
```

Comment:
I put the KCS here too because the article procedure doesn't seem to run if the user aborts the upgrade.  I feel like they might want to look at the KCS article even if they abort.